### ISSUE TITLE
data script

### ISSUE THREAD

**Issue by Nabauu** [hahazh/OFR-BRN#5]
请问为什么vimeo数据集要做复制反转操作，而vid4数据集没有做此操作：
GT_long = torch.cat((GT,torch.flip(GT[:-1],dims = [0]) ),dim=0) 
LR_long = torch.cat((LR_in,torch.flip(LR_in[:-1],dims = [0]) ),dim=0)
LR_mid_long = torch.cat((LR_mid,torch.flip(LR_mid,dims = [0]) ),dim=0)
GT,LR_in,LR_mid = GT_long,LR_long,LR_mid_long

**Comment by hahazh** [comment]
我们这里是参考了basicVSR的做法，也就是更长的序列有助于提升性能。vimeo数据集每个序列只有7张图，如果是时空超分任务，那么输入就只有4张图片，太短了，实际上vimeo可能并不适合作为视频超分的数据集，尤其是测试集（不仅序列短，而且分辨率太低）。而Vid作为经典的视频超分数据集，每个序列都是三四十张图像，这已经足够长了，所以我们就没有再进行flip的操作。

**Comment by Nabauu** [comment]
是的，我同意您“更长的序列有助于提升性能”的说法，但是为了公平对比，是否应该也要测试七张图的序列结果？因为诸如zooming slow-mo和TMnet测试的应该是七帧序列的结果？

**Comment by hahazh** [comment]
您说的很有道理，我们之所以没有重新比较zooming slow-mo和TMnet通常长度的序列是考虑到以下几点：
1.TMNet和zooming slow-mo对于长序列的建模能力并不好，更长的序列并不必然带来性能的提升(参见表2)而我们的方法在使用更长序列时明显有更稳定的提升。这一定程度上是因为我们引入了FRM模块。
2.我们尝试用TMNet和Zooming的预训练权重复现结果，并不能与他们的论文完全对应，甚至在一些数据集（例如Vid上）得到了偏低的结果，这可能与非常多因素有关，例如具体计算PSNR的方式，为了尊重原作者，我们按照习惯直接给出了他们论文中汇报的结果。
3.目前整个时空超分对比都非常混乱，缺乏统一公平的benchmark，一些论文例如VideoINR只用紧邻的两帧来对比其他方法，实际上严重拉低了其他方法的性能，这是非常不公平的，一些方法的训练集也不相同，不管怎么做其实严格来讲都是不公平的。
考虑到以上因素，我们只能选择相对折中的方案，就是如实照抄原作者汇报的结果，希望您能理解。

**Comment by Nabauu** [comment]
感谢您的回复，您说的是有道理的。
